blazar/manager/enforcement.py

```diff
@@ -320,13 +347,16 @@ def check_usage_against_allocation_pre_update(self, reservation_values,
         if not CONF.enforcement.usage_enforcement:
             pass
 
-        project_name = self._get_project_name(lease['project_id'])
-        self.setup_usage_enforcement(project_name)
+        user_name = self._get_user_name(lease['user_id'])
```

Comment:
`user_name` is being redefined here, remove this and simply rely on line 383 setting this variable.